content/en/docs/Writing policies/mutate.md

```diff
@@ -351,6 +351,190 @@ The anchor processing behavior for mutate conditions is as follows:
+
+To define such a policy, trigger resources need to be specified in the `match` block. The target resources, resources that are mutated in the background, are specified in each mutate rule `mutate.targets`. Note that all target resources within a single rule must share the same definition schema. For example, a mutate existing rule fails if this rule mutates both `Pod` and `Deployment` as they do not share the same OpenAPI V3 schema (except `metadata`). 
+
+{{% alert title="Note" color="warning" %}}
+The proper permissions need to be granted to Kyverno service account in order to mutate existing resources. You will need to create a ClusterRole with right permissions and bind it to Kyverno service account via ClusterRoleBinding.
+{{% /alert %}}
+
+
+This policy applies when the trigger resource `staging/disctionary-1` ConfigMap changes, a label `foo=bar` will be added to the target resource `staging/secret-1` Secret.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: "mutate-existing-secret"
+spec:
+  rules:
+    - name: "mutate-secret-on-configmap-event"
+      match:
+        any:
+        - resources:
+            kinds:
+            - ConfigMap
+            names:
+            - dictionary-1
+            namespaces:
+            - staging
+      mutate:
+        targets:
+        - apiVersion: v1
+          kind: Secret
+          name: secret-1
+          namespace: "{{ request.object.metadata.namespace }}"
+        patchStrategicMerge:
+          metadata:
+            labels:
+              foo: bar
+```
+
+By default, the above policy will not be applied when it is installed. This behavior can be configured via `mutateExistingOnPolicyUpdate` attribute. If you set `mutateExistingOnPolicyUpdate` to `true`, Kyverno will mutate the existing secret on policy CREATION and UPDATE.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: "mutate-existing-secret"
+spec:
+  mutateExistingOnPolicyUpdate: true
+  rules:
+    - name: "mutate-secret-on-configmap-event"
+      match:
+        any:
+        - resources:
+            kinds:
+            - ConfigMap
+            names:
+            - dictionary-1
+            namespaces:
+            - staging
+...
+```
+
+### Variables Referencing Target Resources
+
+To reference data in target resources, you can define the variable `target` followed by the path to the desired attribute. For example, using `target.metadata.labels.env` references the label `env` in the target resource.
+
+This policy copies the configmaps' value `target.data.key` to their label with the key `env`.
```

Comment:
"...copies the ConfigMap's value..."

Reply:
The value is actually from target ConfigMaps. 

So it will be -

"...copies the ConfigMaps' value"?